### PR TITLE
Exercise 4

### PR DESCRIPTION
**Pull request by karlysun**
Please enter a word that doesn't exist and have fun :dancer:

### CODE REVIEW

File: exercise-4.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+prompt.start();
+
+prompt.getAsync(['englishWord']).then( function (result) {
+    
+    return result.englishWord;
+        
+}).then( function(userWord){
+    
+    return request('http://words.bighugelabs.com/api/2/11c036dfbb595f828a10c807a507e679/' + userWord+"/json");
```
Comment (ziad-saab) [inline exercise-4.js]: you can put this as the return value of the previous handler

**Comment by ziad-saab** [comment]
@karlysun please check my comment. your code works perfectly though :100:

**Comment by karlysun** [comment]
Update is done as required!

**Comment by ziad-saab** [comment]
:+1:

**Pull request closed without merging**